playbooks/adhoc/uninstall.yml

```diff
@@ -85,7 +85,7 @@
 
   - name: Remove packages
     package: name={{ item }} state=absent
-    when: not is_atomic | bool
+    when: not is_atomic | bool and openshift_remove_all | default(True) | bool
```

Comment:
Instead of applying all of these conditionals to each task, what about grouping them within a block?

Something like this, might clean things up a bit:
```
- block:
  - block:
    - name: Remove packages
      package: name={{ item }} state=absent
      with_items:
      ...
    - name: Remove flannel
      package: name=flannel state=absent
      when: openshift_use_flannel | default(false)
    when: "{{ not is_atomic | bool }}"

  - shell: systemctl reset-failed
    changed_when: False
 
  - shell: systemctl daemon-reload
    changed_when: False
  
  - name: Remove br0 interface
    shell: ovs-vsctl del-br br0
    when: openshift_remove_all | default(True) | bool
    changed_when: False
    failed_when: False
  
  - name: Remove linux interfaces
    shell: ip link del "{{ item }}"
    when: openshift_remove_all | default(True) | bool
    changed_when: False
    failed_when: False
    with_items:
    - lbr0
    - vlinuxbr
    - vovsbr
  when: "{{ openshift_remove_all | default(true) | bool }}"
```